repos/system_upgrade/el7toel8/actors/vimmigrate/actor.py

```diff
@@ -0,0 +1,32 @@
+from leapp.actors import Actor
+from leapp.libraries.actor import library
+from leapp.libraries.common.rpms import has_package
+from leapp.models import Report, InstalledRedHatSignedRPM
+from leapp.tags import ApplicationsPhaseTag, IPUWorkflowTag
+
+
+class VimMigrate(Actor):
+    """
+    Modify Vim configuration files to keep the the same behavior as vim-7.4 has.
```

Comment:
```suggestion
    Modify configuration files Vim X.Y and later to keep the the same behavior as Vim 7.4 and earlier had.
```

Reply:
Please specify **from** which version the configuration changed.

Reply:
Noted.